### DISCUSSION TITLE
dependencyDashboard option not working

### DISCUSSION THREAD

**Discussion by JaykeMeijer** [renovatebot/renovate#9266] [Request Help] [answered]
Hi, I am experimenting with Renovate to manage our dependencies. In the config options I have found the option for the dependency dashboard, which seems interesting. However, when I try to add this, I am getting the following error:

`Invalid configuration option: dependencyDashboard`

I have tried the config preset as well, which gives another error: 

`Error type: Preset name not found within published preset config (:dependencyDashboard)`

I am using Gitlab with the self hosted option as described here: https://simplabs.com/blog/2019/04/24/dependency-updates-for-gitlab/ , ie running the renovate cli through GitLab CI.

My config is below. Am I doing something wrong?

```
{
  "extends": [
    "config:base"
  ],
  "prConcurrentLimit": 5,
  "vulnerabilityAlerts": {
    "labels": ["security"]
  },
  "separateMinorPatch": false,
  "dependencyDashboard": true
}
```

**Comment by viceice** [comment, marked as answer]
You probalby need a newer renovate version. Which version are you using?

Also checkout our runner templates for gitlab [here](https://gitlab.com/renovate-bot/renovate-runner)

**Reply by JaykeMeijer** [reply]
Right, that was the issue. Stupid of me. Thanks!

**Comment by HonkingGoose** [comment]
```json
  "vulnerabilityAlerts": {
    "labels": ["security"]
  },
```

@viceice Does the snippet above work on GitLab?

Renovate docs for `vulnerabilityAlerts` config option says:

> Use this object to customise PRs that are raised when vulnerability alerts are detected (GitHub-only). 

Source: https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts

**Reply by viceice** [reply]
I think they will be ignored on other platforms that github (platform return empty list)

**Reply by viceice** [reply]
I'm pretty sure he uses an outdated renovate version.

**Reply by JaykeMeijer** [reply]
Probably not, as said I am still very much just playing around with it so I copied in some stuff that sounded interesting. As @viceice said my problem was an outdated renovate version.
